frontend/components/molecules/Nav.js

```diff
@@ -7,19 +7,31 @@ import LogoutButton from './LogoutButton';
 import { useRouter } from 'next/router';
 
 const MyNav = styled(SystemNav)`
-  position: fixed;
-  padding-right: ${(props) => props.theme.space.cardMarginSmall}px;
-  padding-top: ${(props) => props.theme.space[4]}px;
-  z-index: 20;
-
+  padding-right: ${(props) => props.theme.space[9]}px;
+  padding-top: ${(props) => props.theme.space[8]}px;
+  padding-bottom: ${(props) => props.theme.space[7]}px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  background-color: ${props => props.theme.colors.greys[0]};
+  shadow: ${props => props.scrolled ? 'default' : 'none'};
   ${(props) => props.theme.mediaQueries.tablet} {
-    right: ${(props) => props.theme.space.cardMargin}px;
```

Comment:
is there a reason why you chose to hardcode a pixel size instead of using theme?

Reply:
This change has been reverted.